### ISSUE TITLE
Folder page ordering doesn't obey 99+

### ISSUE THREAD

**Issue by l3aconator** [getgrav/grav#1524]
The folder numerical ordering doesn't obey the order over 99. Anything over 99 puts them back at the top of the order structure on render instead of continuing sequentially.

**Comment by flaviocopes** [comment]
Shouldn't be the case, just tested. 
You mean in a frontend collection? How is the collection defined? Can you paste the parent page header?

**Comment by l3aconator** [comment]
```
---
title: 'TeamSnap Team Players - Employee Bios'
menu: 'Team Players'
metadata:
    description: 'The talented men and women (and dog!) that work together to bring you TeamSnap products and services.'
content:
    items: '@self.children'
visible: true
---
```
And then all the child pages are labeled numerically do order them and allow for content editors to reorder them the way they want.

**Comment by flaviocopes** [comment]
I assumed you're referring to the order in the frontend, is this right? Or are you talking about the Admin interface?

**Comment by l3aconator** [comment]
The admin interface for content editors shows the order correctly. When it renders on the frontend, it is out of order, with 100+ showing up first and then 1+. So, the list is up to 107. Numbers 100-107 show up at the top of the list and then start counting from 1.

**Comment by flaviocopes** [comment]
I cannot replicate the problem. Can you PM me your user/ folder via Slack? This will make sure I test your same exact code.

**Comment by w00fz** [comment]
I can actually reproduce this on linux only. On mac this is fine.

**Comment by mahagr** [comment]
@flaviocopes Looks like you have overridden `SORT_NATURAL` with `SORT_REGULAR` in `Pages::buildSort()` function. Might have something to do with the wrong ordering.

**Comment by w00fz** [comment]
@l3aconator mind testing my fix in #1564 ?

**Comment by l3aconator** [comment]
@w00fz Will do. Our production servers are linux. I'll take a look.
